src/status_im2/contexts/onboarding/common/carousel/view.cljs

```diff
@@ -0,0 +1,96 @@
+(ns status-im2.contexts.onboarding.common.carousel.view
+  (:require [quo2.core :as quo]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [react-native.navigation :as navigation]
+            [react-native.reanimated :as reanimated]
+            [status-im2.common.resources :as resources]
+            [status-im2.contexts.onboarding.common.carousel.style :as style]
+            [status-im2.contexts.onboarding.common.carousel.animation :as animation]))
+
+(def header-text
```

Comment:
> these header texts are specific to the intro page

Actually they are not specific to intro page, if we delete blurred overlay in any of screens we will see same carousal as intro with headers and background slides 

> so that it can be reused on other screens

With all the connected animations with shared values and weird one 4 screen wide illustrator, I don't think we can create them as separate components. If we want to reuse same animation somewhere, it will be better to copy paste.

Reply:
So, to summarise, all on-boarding screen are three layers (as mentioned on discord)
1. Top layer - screen we are implementing, like app flow, intro screen, login screen
2. Dark black overlay - It is in all screens except intro
3. Last layer is dynamic, for onboarding it has that animated carousel, but for switch profile it has profile below dark overlay.

(So background component, encapsulated last two layers, so that we only need to implement first layer)